third_party/move/move-compiler-v2/src/pipeline/ability_processor.rs

```diff
@@ -509,16 +558,18 @@ impl<'a> Transformer<'a> {
     /// the reason and possible a list of hints is provided as well.
     fn check_ability_for_type(
         &self,
-        id: AttrId,
+        _id: AttrId,
```

Comment:
Can this unused param be actually removed and callers updated?

Reply:
I intentionally keep this in case we need both the location of the bytecode and the location of the argument. Say we want some error message like in "`f(x, y, z)` the argument `z` is dropped".